subsys/bluetooth/audio/Kconfig

```diff
@@ -31,6 +31,15 @@ config BT_AUDIO_TX
 	  This hidden option is enabled when any of the profiles/services
 	  enables support for transmitting of audio data.
 
+config BT_AUDIO_NOTIFY_RETRY_DELAY_MS
+	int "Delay for notification sending retried attempt"
+	range 8 4000
+	default 40
+	help
+	  This option sets the time (in milliseconds) before the stack will
+	  retry to send notification that failed due to lack of TX buffers
+	  available.
```

Comment:
Should these be in unit of 1.25ms so that it uses the same unit as the connection interval (and the interval Kconfigs)? 

With the current, it is not possible to specify a retry for each connection interval when it's e.g. 7.5ms